Files/MultilingualResources/Files.tr-TR.xlf

```diff
@@ -2390,10 +2390,6 @@
           <source>Close tabs to the right</source>
```

Comment:
By the way, can you add .Text to HorizontalMultitaskingControlCloseTabsToTheRight?